### PR TITLE
RM - Add Guide Message 

### PR DESCRIPTION
**Pull request by Diama1**
#1102

### CODE REVIEW

**Comment by Diama1** [comment]
ezgif com-video-to-gif-converter

File: client/src/elm/Backend/ResilienceMessage/Model.elm [hunk 1]
```diff
@@ -17,7 +17,8 @@ type alias ResilienceMessage =
 
 
 type ResilienceCategory
-    = ResilienceCategoryIntroduction
+    = ResilienceCategoryGuide
```
Comment (anvmn) [inline client/src/elm/Backend/ResilienceMessage/Model.elm]: Guide is not a category of message. Guide is constant. Revert and use different approach.

**Review by anvmn — CHANGES_REQUESTED** [review]
@Diama1 we do not need a category 'Guide'.

File: client/src/elm/Pages/MessagingCenter/View.elm [hunk 2]
```diff
@@ -486,7 +490,7 @@ viewResilienceMessage language nurseId nurse model ( messageId, message ) =
             not <| EverySet.member messageId model.expandedMessages
 
         updateTimeRead =
-            model.activeTab == TabUnread
+            model.activeTab == TabGuide
```
Comment (anvmn) [inline client/src/elm/Pages/MessagingCenter/View.elm]: Seems wrong. This is not where you set default active tab.

File: client/src/elm/Pages/MessagingCenter/View.elm [hunk 3]
```diff
@@ -506,7 +510,10 @@ viewResilienceMessage language nurseId nurse model ( messageId, message ) =
         title =
             let
                 messageCategory =
-                    span [ class "category-header" ] [ text <| translate language <| Translate.ResilienceCategory message.category ]
+                    div []
```
Comment (anvmn) [inline client/src/elm/Pages/MessagingCenter/View.elm]: Seems wrong

File: client/src/elm/Pages/MessagingCenter/View.elm [hunk 4]
```diff
@@ -553,6 +560,92 @@ viewResilienceMessage language nurseId nurse model ( messageId, message ) =
         ]
 
 
+viewGuideMessage : Language -> Nurse -> List (Html Msg)
+viewGuideMessage language nurse =
+    [ div [ class "guide-message" ]
+        [ p [ class "title" ]
+            --
```
Comment (anvmn) [inline client/src/elm/Pages/MessagingCenter/View.elm]: remove

File: client/src/elm/Translate.elm [hunk 5]
```diff
@@ -1341,6 +1341,7 @@ type TranslationId
     | ReportCompleted { pending : Int, completed : Int }
     | ResilienceCategory ResilienceCategory
     | ResilienceMessage
+    | ResilienceMessageGuide
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: There are no messages in Guide.
Message is an entity that can be read, set as favourite, etc..
Guide contains sections.

`ResilienceMessageGuide` => `Guide`

File: client/src/elm/Translate.elm [hunk 6]
```diff
@@ -1388,6 +1389,36 @@ type TranslationId
     | ResilienceMessageGrowth4Title
     | ResilienceMessageGrowth4Paragraph1
     | ResilienceMessageGrowth4Paragraph2
+    | ResilienceMessageGuideTitle1
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: This and all bellow => 

`ResilienceMessageGuideXXX` =>  `ResilienceGuideSectionXXX`

File: client/src/elm/Translate.elm [hunk 7]
```diff
@@ -10579,6 +10610,12 @@ translationSet trans =
 
         MessagingTab tab ->
             case tab of
+                TabGuide ->
+                    { english = "Guide"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: => `translationSet Guide`

File: server/hedley/modules/custom/hedley_resilience/hedley_resilience.features.field_base.inc [hunk 8]
```diff
@@ -70,6 +70,7 @@ function hedley_resilience_field_default_field_bases() {
     'module' => 'list',
     'settings' => array(
       'allowed_values' => array(
+        'guide' => 'Guide',
```
Comment (anvmn) [inline server/hedley/modules/custom/hedley_resilience/hedley_resilience.features.field_base.inc]: Not needed

**Review by anvmn — CHANGES_REQUESTED** [review]

**Review by anvmn — APPROVED** [review]

**Pull request merged**